email-service/src/test/java/no/sikt/nva/email/EmailRequestHandlerTest.java

```diff
@@ -52,19 +53,22 @@ public void init() {
                                              "test1@test.no",
                                              "test2@test.no",
                                              "test3.test.no",
-                                             "test4@test.no",
                                              randomString(),
                                              randomString(),
                                              randomString());
     }
 
     @Test
     public void sendsEmailSuccessfully() throws ApiGatewayException {
```

Comment:
when does it send email successfully?

Reply:
Changed name of method.

I think the test if it possible to send mail should perhaps be an integration test? 
Or is it slightly overkill?
I set up APIgateway to check the input body. (and tested it).